compiler/rustc_parse_format/src/lib.rs

```diff
@@ -887,6 +886,23 @@ impl<'a> Parser<'a> {
         }
     }
 
+    fn suggest_format_align(&mut self, alignment: char) {
+        if let Some(pos) = self.consume_pos(alignment) {
+            let pos = self.to_span_index(pos);
+            self.errors.insert(
+                0,
+                ParseError {
+                    description: "expected format parameter to occur after `:`".to_owned(),
+                    note: Some(format!("`{}` comes after `:`.", alignment)),
```

Comment:
Two notes: we don't end sentences in period unless we're printing a paragraph. This note is somewhat redundant with the label, I'd make this `note: None` instead.

Besides that, this is good to merge.

Reply:
Thank you! I have removed the note.